airflow/ui/src/pages/Dashboard/HistoricalMetrics/MetricSection.tsx

```diff
@@ -42,9 +45,11 @@ export const MetricSection = ({ runs, state, total }: MetricSectionProps) => {
     <VStack align="left" gap={1} mb={4} ml={0} pl={0}>
       <Flex justify="space-between">
         <HStack>
-          <StateBadge fontSize="md" state={state}>
-            {runs}
-          </StateBadge>
+          <Link href={`/webapp/${kind}?state=${state}&start_date=${startDate}&end_date=${endDate}`}>
```

Comment:
Let's use the Link from react-router-dom so that we don't have to hardcode `/webapp/` If we want the link to be styled like chakra, we just wrap the RouterLink with the chakra one and pass the param `asChild` (look at our TaskInstances and DagRuns pages for examples`